test/moment-range.js

```diff
@@ -3,7 +3,7 @@
 
 var should    = require('should');
 var moment    = require('moment');
-require('../lib/moment-range');
+require('../dist/moment-range');
```

Comment:
It's been a while since I've used this, but I seem to recall it needed to be `dist`, because it was otherwise importing the main source files, which have things not supported in all node envs, thus causing syntax errors. Granted, this was back over a year ago when the main LTS node version didn't have much ES6 support, and now is largely irrelevant. I'll make the change ;D